lib/apiservers/service/restapi/handlers/decode/networking.go

```diff
@@ -76,9 +77,21 @@ func FromGateway(m *models.Gateway) string {
 func FromImageFetchProxy(p *models.VCHRegistryImageFetchProxy) common.Proxies {
 	http := string(p.HTTP)
 	https := string(p.HTTPS)
+	var nproxy *string
```

Comment:
To be consistent with the other functions in this package (where each function converts one object), this should probably be extracted into its own function (e.g., `fromProxyExclusionList`).

Reply:
noproxy is defined as one field of VCHRegistryImageFetchProxy, so I process it here.